torchao/experimental/CMakeLists.txt

```diff
@@ -21,6 +22,12 @@ if(NOT TORCHAO_INCLUDE_DIRS)
   set(TORCHAO_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/../..)
 endif()
 
+cmake_dependent_option(TORCHAO_BUILD_KLEIDIAI "Download, build, and link against Arm KleidiAI library" ON
```

Comment:
Maybe leave this as cmake arg that gets passed in?

Reply:
sure.